### PR TITLE
Added support for multiple locales per template file

### PR DESCRIPTION
**Pull request by saadb94**
Required by i18n Initiative. 

Multiple locales supported via array per template file. New fn is backward compatible as well for the old style of export. 

If one locale, then template file is named according to that. If multiple, then its a hash of the locales array. Otherwise, if it is * then the template file is named anyLocale. 

Things to consider: 
1. Ordering of the locales array before we hashing, as the hash will change otherwise. 
2. Publishing the node-utils package as we're using a new function added to the crypto library

### CODE REVIEW

File: packages/mdctl-core/streams/section.js [hunk 1]
```diff
@@ -40,6 +40,18 @@ const _ = require('lodash'),
       SectionsCreated = [],
       { privatesAccessor } = require('@medable/mdctl-core-utils/privates')
 
+let hashCode = function (str, seed = 0) {
```
Comment (gastonrobledo) [inline packages/mdctl-core/streams/section.js:43]: you can add this function mdctl-node-utils package https://github.com/Medable/mdctl/blob/master/packages/mdctl-node-utils/crypto.js and use crypto that already exists.

File: packages/mdctl-core/streams/section.js [hunk 2]
```diff
@@ -298,6 +310,55 @@ class ExportSection {
     }
   }
 
+  extractTemplatesGrouped() {
```
Comment (gastonrobledo) [inline packages/mdctl-core/streams/section.js:313]: if extractTemplates is not longer used please remove or use the same method name.

**Review by gastonrobledo — CHANGES_REQUESTED** [review]

File: packages/mdctl-node-utils/crypto.js [hunk 3]
```diff
@@ -22,6 +22,20 @@ function md5FileHash(filename) {
   return hash.digest('hex')
 }
 
+function hashCode(str, seed = 0) {
```
Comment (gastonrobledo) [inline packages/mdctl-node-utils/crypto.js:25]: When I said you could use crypto I mean you could use this 
```
const hash = createHash('md5')
hash.update(str)
return hash.digest('hex')
```

**Review by gastonrobledo — CHANGES_REQUESTED** [review]

File: packages/mdctl-node-utils/crypto.js (hunk 3 above)
Comment (saadb94) [reply packages/mdctl-node-utils/crypto.js:25]: OK, I'll change it to this

**Review by gastonrobledo — APPROVED** [review]

**Review by gastonrobledo — APPROVED** [review]

**Pull request merged**